src/components/genre/Genre.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable camelcase */
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { BsFillArrowRightCircleFill } from 'react-icons/bs';
+import { fetchAllMovies } from '../../redux/MovieSlice';
+import './Genre.css';
+
+const Genre = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const status = useSelector((state) => state.movies.status);
+  const movies = useSelector((state) => state.movies.movies);
+
+  const movide_data = {
+    romance: [],
+    comedy: [],
+    action: [],
+    crime: [],
+    fantasy: [],
+    horror: [],
+  };
+
+  const gotoMovie = (genre) => {
+    // navigate to details page with movie id
+    navigate(`/details/${genre}`);
+  };
+
+  useEffect(() => {
+    if (status === 'idle') {
+      dispatch(fetchAllMovies());
+    }
+  }, [dispatch, status]);
+
+  let content;
```

Comment:
- [ ] Kindly ensure that the home page, which is the genre in your case, has a list of categories that could be filtered using given parameters, e.g. genre name. You can use a search bar to implement the filter.

Reply:
It has been modified